### PR TITLE
fix: #319 Include cpp and cu resources in wheel

### LINKED ISSUE #319: Include cpp and cu resources in wheel
cpp and cu are missing in the wheel file.

### PR DESCRIPTION
**Pull request by hrojek**
# Pull Request      

## What does this PR do?

```Closes #319```       

## Description

**Describe** the changes proposed by this PR below to give the reviewer context below ↓↓↓

```cpp and cu source files were missing in PrimeQA wheel, this change fixes it.```

### CODE REVIEW

**Review by bhavani105 — COMMENTED** [review]
Please update the docs/development.md and a section with instruction on how to build the wheel

**Review by bhavani105 — APPROVED** [review]
LGTM

**Review by franzmpub — APPROVED** [review]
LGTM

File: setup.py [hunk 1]
```diff
@@ -131,4 +131,9 @@
         "Intended Audience :: Science/Research",
         "Programming Language :: Python :: 3",
     ],
+    include_package_data=True,
+    package_data = {
```
[the remark this answers is not in the document]
Comment (hrojek) [reply setup.py]: It worked, thanks!

**Comment by kpfadnis** [comment]
Closing as the work was already covered and merged under `services` release.

**Pull request closed without merging**